params/config.go

```diff
@@ -496,7 +496,7 @@ func (c *ChainConfig) Description() string {
 		banner += fmt.Sprintf(" - Granite:                     @%-10v\n", *c.GraniteTime)
 	}
 	if c.HoloceneTime != nil {
-		banner += fmt.Sprintf(" - Holocene:                    @%-10v\n", *c.HoloceneTime)
+		banner += fmt.Sprintf(" - Holocene:                     @%-10v\n", *c.HoloceneTime)
```

Comment:
What is this diff exactly? It may go away with a rebase if its fixing a spacing issue